src/Components/DefLinear/DefLinear.css

```diff
@@ -0,0 +1,15 @@
+.header {
+    font-size: 3em;
+    font-weight: bold;
+    color: #95CF67;
+}
+
+.Content {
+    font-size: 2em;
+    margin-left: 2em;
+    margin-right: 2em;
+}
+
+.bold {
+    font-weight: bold;
```

Comment:
To avoid extraneous CSS classes, we can leverage Material UI's [Typography](https://material-ui.com/system/typography/).